<launch>
    <node pkg="uavnav" type="test_trj_8" name="note_test_8" output="screen">
        <param name="force_start" type="bool" value="false" />
    </node>
</launch>



